### ISSUE TITLE
Reflections ... Possible Bug ?

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#1438]
Not a show stopper just making you aware.

Using a smooth normal texture and extending the light probe i am able to get the desired reflections that the SSR used to give me however they are not real time like the SSR as the video below shows Move the box and the reflection stays ?

https://youtu.be/yX_M2c6r3Jg

**Comment by plemsoft** [comment]
@synchromesh62 Yes currently we do not added real-time reflection probes , but do have the ability to do this, i will mark it as a feature request for after EA :)
